plugin/klogrus/klogrus.go

```diff
@@ -0,0 +1,63 @@
+package klogrus
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/twmb/franz-go/pkg/kgo"
+)
+
+// Logger provides the kgo.Logger interface for usage in kgo.WithLogger when
+// initializing a client.
+type Logger struct {
+	lr *logrus.Logger
+}
+
+// New returns a new Logger.
+func New(lr *logrus.Logger) *Logger {
+	return &Logger{lr}
+}
+
+// Level is for the kgo.Logger interface.
+func (l *Logger) Level() kgo.LogLevel {
+	return logrusToKgoLevel(l.lr.Level)
```

Comment:
Can this use GetLevel? It looks like Level can be updated atomically with SetLevel